app/models/reports/result.rb

```diff
@@ -85,7 +85,7 @@ def to_s
     end
 
     def last_value(key)
-      self[key].values.last || 0
```

Comment:
I never liked this hacky default here, and we can remove it now.  

Its much better to know if there is actually a value or if its nil.